src/effective-dart/documentation.md

```diff
@@ -32,7 +32,7 @@ generated documentation.
 {:.good}
 <?code-excerpt "docs_good.dart (comments-like-sentences)"?>
 {% prettify dart tag=pre+code %}
-// Not if there is nothing before it.
+// Not if anything comes before it.
```

Comment:
This modified comment seems to no longer match the code sample. It's returning false if nothing comes before it (`_chunks.isEmpty`). The "not" comes from the false, the "nothing" came from the `isEmpty`. 

If you don't like the original comment, I suppose you can change the snippet, perhaps simply:

```
if (_chunks.isNotEmpty) return false;
```

Then the "Not if anything comes before it." will make a bit more sense.